lib/vsc/utils/optcomplete.py

```diff
@@ -164,7 +164,7 @@ def __call__(self, **kwargs):
         if self.CALL_ARGS_OPTIONAL is not None:
             all_args.extend(self.CALL_ARGS_OPTIONAL)
 
-        for arg in kwargs.keys():
+        for arg in list(kwargs.keys()):
```

Comment:
why is the `list()` needed?

Reply:
because we modify `kwargs` by deleting stuff from it while we're looping over it, which results in a crash in Python 3...